app/services/get-capacity-view.js

```diff
@@ -0,0 +1,26 @@
+const getBreadcrumbs = require('../services/get-breadcrumbs')
+const getIndividualWorkloadReports = require('../services/data/get-individual-workload-reports')
+const getWorkloadReports = require('../services/data/get-workload-report-views')
+const routeType = require('../constants/organisation-unit')
+const tableCreator = require('../services/helpers/table-creator')
+const getOrganisationUnit = require('../services/helpers/org-unit-finder')
+
+module.exports = function (id, capacityDateRange, organisationLevel) {
+  var tableType = getOrganisationUnit('name', organisationLevel)
```

Comment:
`tableType` is a bit of a confusing variable name in this context. Perhaps something along the lines of `organisationalUnitType` would be clearer?